tests/lib/totp.js

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!tdd',
+  'intern/chai!assert',
+  'tests/addons/environment',
+  'tests/addons/sinon',
+  'node_modules/otplib/otplib-browser'
+], function (tdd, assert, Environment, sinon, otplib) {
+
+  with (tdd) {
+    suite('totp', function () {
+      var account;
+      var accountHelper;
+      var respond;
+      var client;
+      var RequestMocks;
+      var env = new Environment();
```

Comment:
Is it necessary to initialize `env` here, if we assign it a value in `beforeEach`?